charts/px-monitor/templates/cortex/cortex-backend.yaml

```diff
@@ -155,6 +155,10 @@ spec:
               value: rack1
             - name: CASSANDRA_ENABLE_RPC
               value: "true"
+            - name: MAX_HEAP_SIZE
```

Comment:
we should have a check here {{- if   .Values.cassandra.jvm.maxHeapSize}} for both

Reply:
Done.